clients/admin-ui/src/features/system/dictionary-data-uses/DataUseCheckboxTable.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Checkbox,
+  Table,
+  Tag,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@fidesui/react";
+
+import { DataUse } from "../../../types/api";
+import { DictDataUse } from "../../plus/types";
+
+interface Props {
+  onChange: (dataUses: DictDataUse[]) => void;
+  allDataUses: DataUse[];
+  dictDataUses: DictDataUse[];
+  checked: DictDataUse[];
+}
+
+const DataUseCheckboxTable = ({
+  onChange,
+  allDataUses,
+  dictDataUses,
+  checked,
+}: Props) => {
+  const handleChangeAll = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.checked) {
+      onChange(dictDataUses);
+    } else {
+      onChange([]);
+    }
+  };
+
+  const onCheck = (dataUse: DictDataUse) => {
+    const exists =
+      checked.filter((du) => du.data_use === dataUse.data_use).length > 0;
+    if (!exists) {
+      const newChecked = [...checked, dataUse];
+      onChange(newChecked);
+    } else {
+      const newChecked = checked.filter(
+        (use) => use.data_use !== dataUse.data_use
+      );
+      onChange(newChecked);
+    }
+  };
+
+  const declarationTitle = (declaration: DictDataUse) => {
+    const dataUse = allDataUses.filter(
+      (du) => du.fides_key === declaration.data_use
+    )[0];
+    if (dataUse) {
+      return dataUse.name;
+    }
+    return declaration.data_use;
+  };
+
+  const allChecked = dictDataUses.length === checked.length;
+
+  return (
+    <Table variant="unstyled" size="sm" border="1px" borderColor="gray.200">
+      <Thead border="1px" borderColor="gray.200" backgroundColor="gray.50">
+        <Tr sx={{ "border-collapse": "separate" }}>
```

Comment:
```suggestion
        <Tr borderCollapse="separate">
```

I got this error in the console. I think this is what it's asking for

![Screenshot from 2023-09-07 13-41-41](https://github.com/ethyca/fides/assets/17103888/19a5c62f-68e2-493f-96be-58a7802e26ed)

Reply:
if that doesn't work then `styte={{"border-collapse": "separate"}}` might work as well

Reply:
Oops, yeah, I left that in from when I was testing some stuff before, it's not doing anything.